dropwizard-parent-pom/pom.xml

```diff
@@ -252,11 +251,6 @@
       </dependency>
       <dependency>
         <groupId>io.dropwizard</groupId>
-        <artifactId>dropwizard-spdy</artifactId>
```

Comment:
@bbeck : what does this approach mean for the exclusions that were added to io.dropwizard:dropwizard-core?  (findbugs and javax.inject were excluded)